examples/tests/valid/dynamo.w

```diff
@@ -0,0 +1,94 @@
+bring "@cdktf/provider-aws" as tfaws;
+bring aws;
+bring cloud;
+bring util;
+
+enum AttributeType {
+  String,
+  Number,
+  Binary,
+}
+
+struct Attribute {
+  type: AttributeType;
+  value: Json;
+}
+
+class DynamoTable {
+  table: tfaws.dynamodbTable.DynamodbTable;
+  tableName: str;
+  init() {
+    let target = util.env("WING_TARGET");
+    if target != "tf-aws" {
+      throw("Unsupported target: ${target} (expected 'tf-aws')");
+    }
+
+    this.table = new tfaws.dynamodbTable.DynamodbTable(
+      name: "Default",
+      billingMode: "PAY_PER_REQUEST",
+      hashKey: "Flavor",
+      attribute: [
+        {
+          name: "Flavor",
+          type: "S",
+        },
+      ],
+    );
+    this.tableName = this.table.name;
+  }
+
+  _bind(host: std.IInflightHost, ops: Array<str>) {
+    if let host = aws.AwsFunction.from(host) {
```

Comment:
Alternative API naming (thoughts?)
```suggestion
    if let host = aws.Hosts.asAwsFunction(host) {
```

Reply:
```
aws.Function.from(host)
```

Not sure we need the Aws prefix here. Ha?